README.md

```diff
@@ -1,19 +1,44 @@
-# Git New
+# Generic New Project Template
 
-_A cloneable Git template for repeatable and consistent project bootstraps_
+_A Git template for bootstrapping a variety of projects with a consistent set of control files._
 
-# Table of Contents
+## Instructions
 
-- [Git New](#git-new)
-- [Table of Contents](#table-of-contents)
+1. Create your new repository by clicking the "Use this template" [button](https://github.com/tforster/git-new/generate) from the GitHub repository.
```

Comment:
This goes on the assumption that the GitHub UI will be utilized every time.  Should we have separate, more generic steps for the first few?

Reply:
I thought about that and my rationale was that now *this* repo has been set up as a GitHub template, it implies it is being used from GitHub. And templates themselves, are GitHub forks wrapped in some UI sugar. Of note, forking is not part of the Git spec and not in the Git cli.

How about 

```suggestion
1. Create your new repository by clicking the "Use this template" [button](https://github.com/tforster/git-new/generate) from the GitHub repository. (or download from https://github.com/tforster/git-new/archive/master.zip and unzip)
```

Reply:
That makes sense.  Does the template forking route associate the repos the way regular forking does?  Haven't gone through the flow yet.

Reply:
I suspect there is some difference and we should compare both at some point to fully understand. 

For process going related to inline suggestions like above, should the suggestion author commit, or should the suggestion reviewer commit? I'm assuming you see the same button I do to "Commit suggestion"

Reply:
I see that button and also a greyed out button to `Add suggestion to batch`.

Reply:
I think the person on the receiving end of the suggestion, regardless of whom authored the parent PR should be the one to Commit the suggestion, or explicitly reject it via a comment. Thoughts?

Reply:
Makes sense to me.